src/app/lib/views/movie_detail.js

```diff
@@ -7,7 +7,8 @@
 
         ui: {
             bookmarkIcon: '.favourites-toggle',
-            watchedIcon: '.watched-toggle'
+            watchedIcon: '.watched-toggle',
+            selected_lang: '.selected-lang'
```

Comment:
I'm backporting from forks, it does make things better than without it (hence why they added it)